app/assets/javascripts/components/widgets_box.js.jsx

```diff
@@ -68,40 +68,54 @@ var Widget = React.createClass({
   },
 
   render(){
-    var rawMarkup = marked(this.props.children.toString(), {sanitize: true});
```

Comment:
FWIW, at a general level, it's safe to assume that our campaigners are willing to at least try to get to know Markdown. I think it's possible we might instead want to go to a more traditional WYSIWYG editor, but Markdown is at least on the table and we could include it in order to solicit feedback.